src/components/ha-card.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  css,
+  CSSResult,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+class HaCard extends LitElement {
+  @property() public header?: {};
```

Comment:
```suggestion
  @property() public header?: string;
```

Reply:
Right. Leftover from when I still tried to get more advanced headers to work... like litHtml templates with toggles.

Reply:
`string | TemplateResult` would be the correct type then 👍

Reply:
However, that should be done using named slots.